src/control/cmd/daos/snapshot.go

```diff
@@ -211,6 +215,32 @@ func listContainerSnapshots(ap *C.struct_cmd_args_s, containerID string) ([]*sna
 	return snapshots, nil
 }
 
+func printSnaps(out io.Writer, snaps []*snapshot) {
```

Comment:
This function looks good, but would be nice to have a unit test for this output. I would suggest shifting this function into the cmd/daos/pretty subpackage and reference the existing unit tests for verifying output.